organizing_hub/templates/calls/callcampaign_form.html

```diff
@@ -10,7 +10,7 @@
 {% endblock %}
 
 {% block content_heading %}
-Create Campaign
+{% if update_view %}Update{% else %}Create{% endif %} Campaign
```

Comment:
Could also put campaign title here for update?

Reply:
Ok nevermind, I see now there is an editable title field.

Reply:
👍